source/server/http/admin.cc

```diff
@@ -317,7 +321,7 @@ AdminImpl::AdminImpl(const std::string& access_log_path, Network::Address::Insta
           {"/server_info", "print server version/status information",
            MAKE_HANDLER(handlerServerInfo)},
           {"/stats", "print server stats", MAKE_HANDLER(handlerStats)}} {
-
+  profiler_path_ = profiler_path;
```

Comment:
It might be nicer to initialize this in the above constructor member initialization list, i.e.

``` : server_(server), ..., profiler_path_(profiler_path)```